### PR TITLE
feat: integrate exchange contracts

### LINKED ISSUE #322: [FEATURE] Integrating Exchange Contracts
### 1. Is your feature request related to a problem? Please describe.

- There are many common functions among Exchange contracts, which can be integrated.
- By implementing common functions as a single contract, maintainability is expected to be improved.

### 2. Describe the solution you'd like

- Remove support for Exchange features that are not currently officially supported (OTCExchange).
- Integrate Exchange contracts.

### PR DESCRIPTION
**Pull request by YoshihitoAso**
close #322

### CODE REVIEW

File: contracts/IbetStandardToken.sol [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+
+pragma solidity ^0.8.0;
+
+import "./SafeMath.sol";
+import "./Ownable.sol";
+import "../interfaces/ContractReceiver.sol";
+import "../interfaces/IbetStandardTokenInterface.sol";
+
+
+/// @title ibet Coupon Token
```
Comment (urushibata-k) [inline contracts/IbetStandardToken.sol:28]: invalid comment

**Comment by urushibata-k** [comment]
Need to fix contract_deploy.py

File: tests/test_IbetExchange.py [hunk 2]
```diff
@@ -78,54 +75,54 @@ class TestTokenFallback:
...
     # Normal_1
-    def test_normal_1(self, users, membership_exchange):
+    def test_normal_1(self, users, exchange):
         _issuer = users['issuer']
         _value = 100
 
         # issue token
-        deploy_args = init_args(membership_exchange.address)
+        deploy_args = init_args(exchange.address)
         membership_token = deploy(users, deploy_args)
```
Comment (urushibata-k) [inline tests/test_IbetExchange.py:84]: this is standard token(Same for other tests)

**Comment by urushibata-k** [comment]
https://github.com/BoostryJP/ibet-SmartContract/blob/a8b7f0b413e77f30d18618f80ebadf76b8e46040/contracts/IbetExchange.sol#L529

https://github.com/BoostryJP/ibet-SmartContract/blob/dev-21.9.0/contracts/IbetCouponExchange.sol#L528

Previously IbetCouponExchange did not have this process, is it as expected?(Bug from before version?)

File: contracts/IbetStandardToken.sol (hunk 1 above)
Comment (YoshihitoAso) [reply contracts/IbetStandardToken.sol]: That's right. I have corrected it.

File: tests/test_IbetExchange.py (hunk 2 above)
Comment (YoshihitoAso) [reply tests/test_IbetExchange.py]: That's right. I have corrected it.

**Comment by YoshihitoAso** [comment]
> https://github.com/BoostryJP/ibet-SmartContract/blob/a8b7f0b413e77f30d18618f80ebadf76b8e46040/contracts/IbetExchange.sol#L529
> 
> https://github.com/BoostryJP/ibet-SmartContract/blob/dev-21.9.0/contracts/IbetCouponExchange.sol#L528
> 
> Previously IbetCouponExchange did not have this process, is it as expected?(Bug from before version?)

I do not think this check is necessary. It has been removed.
As for IbetCouponExchange, I had removed it in the last modification, but it had caused a degrade.

**Pull request merged**